tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_lstm.py

```diff
@@ -375,290 +311,12 @@ def check_forward(
-            functions.n_step_bilstm(
-                self.n_layers, self.dropout, h, c, ws, bs, xs)
-
-    def test_inconsistent_input_size_cpu(self):
-        x_in_size = 4  # inconsistent in_size with that of ws.
-        xs = [numpy.random.uniform(-1, 1, (b, x_in_size)).astype('f')
-              for b in self.batches]
-        self.check_inconsistent_input_size(
-            self.hx, self.cx, xs, self.ws, self.bs)
-
-    def check_inconsistent_input_size_gpu(self, use_cudnn):
-        x_in_size = 4  # inconsistent in_size with that of ws.
-        xs = [numpy.random.uniform(-1, 1, (b, x_in_size)).astype('f')
-              for b in self.batches]
-
-        hx = _to_gpu(self.hx)
-        cx = _to_gpu(self.cx)
-        xs = _to_gpu(xs)
-        ws = _to_gpu(self.ws)
-        bs = _to_gpu(self.bs)
-        with chainer.using_config('use_cudnn', use_cudnn):
-            self.check_inconsistent_input_size(hx, cx, xs, ws, bs)
-
-    @attr.gpu
-    def test_inconsistent_input_size_gpu_cudnn_always(self):
-        self.check_inconsistent_input_size_gpu('always')
-
-    @attr.gpu
-    def test_inconsistent_input_size_gpu_cudnn_never(self):
-        self.check_inconsistent_input_size_gpu('never')
-
-
-def _stack_weight(ws):
-    # TODO(unno): Input of the current LSTM implementation is shuffled
-    w = functions.stack(ws, axis=1)
-    shape = w.shape
-    return functions.reshape(w, (shape[0] * shape[1],) + shape[2:])
-
-
-def count_close(x, y, atol=1e-4):
-    assert x.shape == y.shape
-    return int(sum(abs(x - y) / abs(x) < atol))
-
-
-def lstm_without_dropout(n_layer, dropout, hx, cx, ws, bs, xs):
-    xws = [_stack_weight([w[2], w[0], w[1], w[3]]) for w in ws]
-    hws = [_stack_weight([w[6], w[4], w[5], w[7]]) for w in ws]
-    xbs = [_stack_weight([b[2], b[0], b[1], b[3]]) for b in bs]
-    hbs = [_stack_weight([b[6], b[4], b[5], b[7]]) for b in bs]
-    xs = [xs[i] for i in range(3)]
-    ys = []
-    for x in xs:
-        cx_next = []
-        hx_next = []
-        for layer in range(n_layer):
-            c = cx[layer]
-            h = hx[layer]
-
-            if layer != 0:
-                # Only multiply ratio
-                x = x * (1 / (1.0 - dropout))
-            lstm_in = functions.linear(x, xws[layer], xbs[layer]) + \
-                functions.linear(h, hws[layer], hbs[layer])
-            c_new, h_new = functions.lstm(c, lstm_in)
-            cx_next.append(c_new)
-            hx_next.append(h_new)
-            x = h_new
-        cx = cx_next
-        hx = hx_next
-        ys.append(x)
-    cy = functions.stack(cx)
-    hy = functions.stack(hx)
-    return hy, cy, ys
-
-
-def rand_vector(shape):
-    # return cuda.cupy.random.randint(-2, 2, shape).astype('f')
-    return cuda.cupy.random.uniform(-1, 1, shape).astype('f')
-    # return cuda.cupy.ones(shape).astype('f')
-
-
-@testing.parameterize(*testing.product({
-    'use_cudnn': ['always', 'auto', 'never'],
-}))
-@attr.cudnn
-class TestNStepLSTMDropout(unittest.TestCase):
```

Comment:
Is removing test for `dropout = 0.3` intended?

Reply:
Yes, you're right, I had removed it by mistake, I have added it back now.